sensor_msgs/Image           input_image
---
geometry_msgs/PoseArray     centroid